index/ask/pb/ask.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+package filecoin.ask.pb;
+
+option java_multiple_files = true;
+option java_package = "io.textile.filecoin.ask.pb";
+option java_outer_classname = "FilecoinAsk";
+option objc_class_prefix = "TTE";
+
+message Query {
+    uint64 maxPrice = 1;
+	uint64 pieceSize = 2;
+	int32 limit = 3;
+	int32 offset = 4;
+}
+
+message StorageAsk {
+	uint64 price = 1;
+	uint64 minPieceSize = 2;
+	string miner = 3;
+	uint64 timestamp = 4;
+	uint64 expiry = 5;
+}
+
+message Index {
+    int64 lastUpdated = 1;
+    uint64 storageMedianPrice = 2;
+    map<string, StorageAsk> storage = 3;
+}
+
+message GetRequest {
+}
+
+message GetReply {
+    Index index = 1;
+}
+
+message QueryRequest {
+    Query query = 1;
+}
+
+message QueryReply {
+    repeated StorageAsk asks = 1;
+}
+
+service API {
+    rpc Get(GetRequest) returns (GetReply) {}
+    rpc Query(QueryRequest) returns (QueryReply) {}
+}
```

Comment:
Went ahead and made a service for the ask index. It's currently mixed in with the deals service. I plan on removing asks service code from the deals service in an upcoming PR.

Reply:
:+1:

Reply:
:+1: